### ISSUE TITLE
new rule for empty blank lines

### ISSUE THREAD

**Issue by evilebottnawi** [eslint/eslint#4290]
```javascript
var a = 10;


var b = 11;
```
Add rule for blank empty lines with options max-blank-lines. What do you think guys?

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by nzakas** [comment]
Already have it: http://eslint.org/docs/rules/no-multiple-empty-lines
